resources/guides/ta_guide.md

```diff
@@ -0,0 +1,99 @@
+## What is a TA?
+
+A TA is a unionized student position where students can work with instructors to run University of Windsor courses. Instructors could ask for a TA's help in a variety of ways:
+
+- Holding Office Hours 
+- Grade student assessments
+- Proctor Exams
+- Taking notes or answering questions during classes
+    
+A TA refers to an undergraduate student while GA (Graduate Assistant) refers to Graduate students
+
+## Why should you become a TA?
+
+Being a TA provides many benefits:
+
+- Work experience with a subject of interest
+- Create connections with Professors and Instructors
+- Paid Position
+- Further your own learning on the subject
+    
+## How to become a TA?
+
+To become a TA a student _typically_ needs to have completed their first year of courses. 
+
+The processes will **usually** begin a few months before the start of the next term:
+
+| Term   | TA Process Begins | Staring Month |
+|:------:|:-----------------:|:-------------:|
+| Winter | October           | January       |
+| Spring | March             | May           |
+| Fall   | July              | September     |
+
+The processes starts with an email from the department explaining which courses will require TA positions and how many positions are available. This email will cover which courses will have positions available and basic information about the positions like:
+
+- All courses and labs will be on campus (including some Distance Education courses that may require on campus office hours).
+- All GA/TA employees must make themselves available, and must be available for all assigned duties, both on-campus and online as required.
+- Each GA position is expected to be for a total of 140 hours. A limited number of GA positions may be available for fewer hours.
+- The total number of hours of a TA appointment will depend on the number of applications received and is anticipated to be 75-100 hours per position.  Note that Teaching Assistants can be employed for no more than 100 TA hours per semester total.
+
+
+Once confirming that you meet the basic requirements for eligibility:
+
+- Successful completion of the course (80% minimum average) or demonstrated experience/knowledge in subject matter (which will be determined by interview). 
+- Must have a minimum cumulative average of 75%. Must have a minimum major average of 80%. 
+- Must have excellent communication skills. Brightspace/Teams experience
+- Must be familiar with Brightspace and MS Teams or be able to acquire knowledge through training.  
+- Remote internet access requirements: Must have access to a computer with a reliable internet connection with video and audio capabilities. 
+    
+Or reviewing to see if you meet the preferred requirements:
+
+- Preferred qualifications will only be considered if there are 2 or more applicants who meet the required essential qualifications and who are relatively equal based on the criteria as per articles 12:03 or 12:04 of the CUPE 4580 collective agreement. 
```

Comment:
Are the articles 12:03 and 12:04, or 12.03 and 12.04? I've never seen colon-separated articles before.

Reply:
The collective agreement uses colons for some reason, I agree 12.03 looks better than 12:03 but I think sticking with the way they organize it is best overall

Reply:
Alrighty. Didn't know if it was a typo or just how they do it. Thanks!